tests/ttnn/unit_tests/operations/test_typecast.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+
+import ttnn
+import tt_lib
+from models.utility_functions import is_grayskull
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from models.utility_functions import torch_random
+
+
+# The idea of the test is to convert bfloat16 to uint32 into preallocated uint32 tensor
+@pytest.mark.skipif(is_grayskull(), reason="GS does not support fp32/uint32/uint16 data types")
+def test_typecast_output_tensor(device):
+    torch.manual_seed(0)
+
+    h = w = 32
+    from_dtype = ttnn.DataType.BFLOAT16
```

Comment:
Please use `ttnn.bfloat16`. It's an alias for `ttnn.DataType.BFLOAT16` but looks much nicer IMO. Same comment about other places where `ttnn.DataType.<...>` is used

Reply:
Awesome! Thank you!